src/stores/RoomListStore.tsx

```diff
@@ -28,14 +28,17 @@ export class RoomListStore {
   private readonly _roomService: RoomService = new RoomService();
   readonly rootStore: RootStore;
   private _createdRoomTitle: string = "";
-
+  private _selectedImageFile?: File;
+  private _imgUrl: string = "";
```

Comment:
선택된 이미지가 없는경우는 undefined가 좋을 것같아요. 값이 없음을 확실히 나타내기 때문입니다.
그리고 필드 이름도 가능한 줄임말 없이 나타내는게 좋을 것 같아요.

아 참고로 이 값이 observable이려면 초기에 값이 존재해야합니다. 아래처럼요
```suggestion
  private _selectedImageUrl?: string = undefined;
```